sympy/geometry/polygon.py

```diff
@@ -400,8 +401,43 @@ def centroid(self):
             v = x1*y2 - x2*y1
             cx += v*(x1 + x2)
             cy += v*(y1 + y2)
-        return Point(simplify(A*cx), simplify(A*cy))
-
+        return Point(simplify(A*cx), simplify(A*cy))   
+    def second_moment_of_area(self,p=None):
+        """Second moment of area of polygon
+	    RETURNS
+	    =======
+	    I_xx,I_yy (about the centroid if point is not given othervise about the given point)
+	    Example
+        =======
+        >>> from sympy import *
+        >>> a,b=symbols('a,b')
+        >>> p1,p2,p3,p4,p5=[(0,0),(a,0),(a,b),(0,b),(a/3,b/3)]
+        >>> rectangle=Polygon(p1,p2,p3,p4)
+        >>> rectangle.second_moment_of_area()
+        (a*b**3/12, a**3*b/12)
+        >>> rectangle.second_moment_of_area(p5)
+        (a*b**3/9, a**3*b/9)
+        """
```

Comment:
Review this document: https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt to see how to format the docstrings.